NotadogApi/Domain/Repositories/UserRepository.cs

```diff
@@ -13,9 +13,8 @@ public UserRepository(AppDbContext context) : base(context)
         {
         }
 
-        public async Task<IEnumerable<User>> GetAllAsync()
-        {
-            return await _context.Users.ToListAsync();
-        }
+        public async Task<IEnumerable<User>> GetAllAsync() => await _context.Users.ToListAsync();
```

Comment:
If you return List object, then it is better to change return type to List or not to enumerate. Otherwise it will enumerate twice (here and on the ASP layer). See https://stackoverflow.com/questions/472669/freely-convert-between-listt-and-ienumerablet#comment290277_472671.